producer/proto/render.go

```diff
@@ -61,11 +61,152 @@ var (
 		0x86dd: "IPv6",
 	}
 	protoName = map[uint32]string{
+		0:   "HOPOPT",
 		1:   "ICMP",
+		2:   "IGMP",
+		3:   "GGP",
+		4:   "IPv4",
+		5:   "ST",
 		6:   "TCP",
+		7:   "CBT",
+		8:   "EGP",
+		9:   "IGP",
+		10:  "BBN-RCC-MON",
+		11:  "NVP-II",
+		12:  "PUP",
+		13:  "ARGUS",
+		14:  "EMCON",
+		15:  "XNET",
+		16:  "CHAOS",
 		17:  "UDP",
-		58:  "ICMPv6",
+		18:  "MUX",
+		19:  "DCN-MEAS",
+		20:  "HMP",
+		21:  "PRM",
+		22:  "XNS-IDP",
+		23:  "TRUNK-1",
+		24:  "TRUNK-2",
+		25:  "LEAF-1",
+		26:  "LEAF-2",
+		27:  "RDP",
+		28:  "IRTP",
+		29:  "ISO-TP4",
+		30:  "NETBLT",
+		31:  "MFE-NSP",
+		32:  "MERIT-INP",
+		33:  "DCCP",
+		34:  "3PC",
+		35:  "IDPR",
+		36:  "XTP",
+		37:  "DDP",
+		38:  "IDPR-CMTP",
+		39:  "TP++",
+		40:  "IL",
+		41:  "IPv6",
+		42:  "SDRP",
+		43:  "IPv6-Route",
+		44:  "IPv6-Frag",
+		45:  "IDRP",
+		46:  "RSVP",
+		47:  "GRE",
+		48:  "DSR",
+		49:  "BNA",
+		50:  "ESP",
+		51:  "AH",
+		52:  "I-NLSP",
+		53:  "SWIPE",
+		54:  "NARP",
+		55:  "Min-IPv4",
+		56:  "TLSP",
+		57:  "SKIP",
+		58:  "IPv6-ICMP",
```

Comment:
Would rather keep ICMPv6 since it's already being used

Reply:
see comment above

Reply:
While I agree it's IANA, I would prefer not breaking existing dashboads

Reply:
my 2 cents:
- I dont think we can mitigate the absence of information without some kind of breakage of existing implementations. As the Protocol Name is the only hint to the protocol field of the actually sampled packet (when utilizing JSON output) I'd prefer IANA names. 
- imho a viable alternative would be to add a field to the json output (proto_num?) with the int32 protonum - which would allow a clear mapping to official protocol names (for example in an enricher) and wont break existing implementations relying on the strings
I'd keep all other protocols in protoName with their official names - with "ICMPv6" as only deviation.
@lspgn @marioschaefer wdyt

Reply:
@DerInti 
point 1:  
i'm on the complete same page

point 2:  
I also thought about that and its probably the cleanest solution, bcs you have the number and the name.  
But this will also break the existing implementation, since we will stick to the IANA-Names and the existing implementation doesn't use them.
on the other hand: how much longer do we want to carry the wrong name around with us. sooner or later it will be changed. the longer you wait, the more it hurts. that's why i'm a big fan of "sooner rather than later".